__mocks__/forms/omrs-forms/afe-schema-trasformer-form.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
@arodidev I think you need to do the rebase correctly. I'm worried about these unintended changes.

Reply:
FIxing this, sorry about the mishaps.